app/views/shared/check_answers/_item.html.erb

```diff
@@ -13,7 +13,7 @@
   </dd>
   <% unless read_only %>
     <dd class="govuk-summary-list__actions">
-      <% unless read_only %>
+      <% unless read_only || url == :show_blank_action %>
```

Comment:
Is this `unless` needed given there's one on line 14? If so should they both have the same conditions? What happens if `read_only == false` and `url == :show_blank_action`? It looks like we would display the `govuk-summary-list` but not the `change` link - is that what's intended?

Reply:
Yes it is to keep the spacing/styling, but you are right the 2nd read only is redundant.